microsoft-edge/devtools-guide-chromium/remote-debugging/windows.md

```diff
@@ -179,10 +195,10 @@ Notes about screencasts:
 <!-- ====================================================================== -->
 ## Known issues<!-- todo: update -->
 
-* The **Event Listeners** pane in the **Elements** tool is blank on Windows 10 version 1903.  The team plans to fix the **Event Listeners** pane in a servicing update to Windows 10 version 1903.
+* The **Event Listeners** pane in the **Elements** tool is blank on Windows 10 version 1903.  The team plans to fix the **Event Listeners** pane in a servicing update to Windows 10 version 1903.<!-- todo: done? -->
 
-* The **Cookies** pane in the **Application** panel is blank on Windows 10 version 1903.  The team plans to fix the **Cookies** pane in a service update to Windows 10 version 1903.
+* The **Cookies** pane in the **Application** panel is blank on Windows 10 version 1903.  The team plans to fix the **Cookies** pane in a service update to Windows 10 version 1903.<!-- todo: done? -->
```

Comment:
is **Cookies** pane fixed, in a service update to Windows 10 version 1903?
@captainbrosset

Reply:
Windows 10 end of support is coming in October 2025. I don't have a quick way of testing this particular issue, but I suggest deleting the "Known issues" section altogether.